lib/hint/rubocop_style/version.rb

```diff
@@ -1,5 +1,5 @@
 module Hint
   module RubocopStyle
-    VERSION = '0.2.0'.freeze
+    VERSION = '0.3.0'.freeze
```

Comment:
Wonder if we should be bumping minor version for these tweaks? Guess it depends on how many we have :)